### PR TITLE
MessageBar: Semantic colors for default and success cases.

### LINKED ISSUE #11603: Success and Info Icon colors are not easily customizible
Looking at https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts we can see that the icons for some types of MessageBar can be customized, but for Success and Info, we need to change green or neutralSecondary colors from the palette, which can affect a lot more than just icons.

Is it possible to have semanticColors for these as well?
 
### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: High

Products/sites affected: Azure for Machine Learning

### PR DESCRIPTION
**Pull request by jdhuntington**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #11603
- [X] Include a change request file using `$ yarn change`

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11637)

### CODE REVIEW

**Comment by ecraig12345** [comment]
Makes sense to me, but you should definitely get someone who's more familiar with semantic slots (and the process for adding them) to review.

File: packages/styling/src/styles/theme.ts [hunk 1]
```diff
@@ -276,14 +276,16 @@ function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean, depCom
...
+    infoBackground: p.neutralLighter,
+    infoText: p.neutralSecondary,
+    successBackground: !isInverted ? 'rgba(95, 210, 85, .2)' : 'rgba(186, 216, 10, .4)',
     successText: !isInverted ? '#107C10' : '#92c353',
-    errorBackground: !isInverted ? 'rgba(245, 135, 145, .2)' : 'rgba(232, 17, 35, .5)',
-    blockingBackground: !isInverted ? 'rgba(250, 65, 0, .2)' : 'rgba(234, 67, 0, .5)',
     warningBackground: !isInverted ? 'rgba(255, 200, 10, .2)' : 'rgba(255, 251, 0, .6)',
     warningHighlight: !isInverted ? '#ffb900' : '#fff100',
-    successBackground: !isInverted ? 'rgba(95, 210, 85, .2)' : 'rgba(186, 216, 10, .4)',
+    warningText: !isInverted ? '#333333' : '#ffffff',
```
Comment (aneeshack4) [inline packages/styling/src/styles/theme.ts]: This is just being moved for alphabetization sake right?

**Comment by aneeshack4** [comment]
This build failure's strange - there isn't much information in the log

File: packages/styling/src/interfaces/ISemanticColors.ts [hunk 2]
```diff
@@ -158,6 +158,11 @@ export interface ISemanticColors extends ISemanticTextColors {
    */
   successBackground: string;
 
+  /**
+   * Background for info
```
Comment (phkuo) [inline packages/styling/src/interfaces/ISemanticColors.ts]: be more descriptive here

**Comment by ecraig12345** [comment]
It's failing to install Node for some reason. @JasonGore or @kenotron do you know why this might be happening?
image

**Comment by JasonGore** [comment]
Not sure, that task hasn't changed recently, and didn't change with my PR. There are two newer builds that installed Node fine. Perhaps the download failed for some reason. I'll try rerunning it again.

**Comment by JasonGore** [comment]
Got past Node the second time. 🤷‍♂️

File: packages/styling/src/styles/theme.ts (hunk 1 above)
Comment (jdhuntington) [reply packages/styling/src/styles/theme.ts]: Yes! Wanted to make sure each value had a pair.

**Comment by aneeshack4** [comment]
@ecraig12345 @dzearing looks like this has been waiting for your review for a while - blocking #11603 from being closed

**Comment by ecraig12345** [comment]
Do we still need this? Also not sure how it would integrate with @chrismohr's PR #12821. cc @phkuo

**Comment by phkuo** [comment]
The linked issue was fixed by Chris' PR.

**Pull request closed without merging**